src/main/java/uk/gov/hmcts/darts/common/repository/MediaRepository.java

```diff
@@ -32,9 +32,11 @@ public interface MediaRepository extends JpaRepository<MediaEntity, Long>,
            JOIN he.medias me
            WHERE he.id = :hearingId
            AND me.isCurrent = true
+           AND (me.isHidden = false or me.isHidden = :includeHidden)
```

Comment:
Formatting suggestion
```suggestion
           AND (me.isHidden = false OR me.isHidden = :includeHidden)
```

Reply:
Agreed applied